app/Console/Kernel.php

```diff
@@ -32,6 +32,6 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        //
+        $schedule->command('queue:work --sleep=3 --tries=3')->everyMinute();
```

Comment:
I just want to make it easiest for the user - but if it's a technical issue, then I'm all for not doing it too :)